airflow/models/xcom_arg.py

```diff
@@ -196,7 +191,10 @@ def get_task_map_length(self, run_id: str, *, session: Session) -> int | None:
     def resolve(self, context: Context, session: Session = NEW_SESSION) -> Any:
         """Pull XCom value.
 
-        This should only be called during ``op.execute()`` in respectable context.
+        This should only be called during ``op.execute()`` in respectable
+        context. Note that although the ``ResolveMixin`` parent mixin also has a
```

Comment:
`respectable context` could probably benefit from clearer wording. though i realize this was already here.